<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="SOAP APIs"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="0.792">
  <properties />
            <testcase name="TestPostMethod" time="0.792" classname="Phase3.RestAssuredLearningPractice.PostRequestForCountryCode">
                <failure type="java.lang.AssertionError"
                  message="1 expectation failed.
Expected status code &lt;200&gt; but was &lt;500&gt;.
"
              >
<![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <500>.

  java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
  java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
  java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
  org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
  org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
  org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:57)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
  io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
  io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
  io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
  io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
  java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
  java.base/java.lang.reflect.Method.invoke(Method.java:580)
  org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
  org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
  org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
  org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
  io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:135)
  io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
  io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:143)
  io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
  Phase3.RestAssuredLearningPractice.PostRequestForCountryCode.TestPostMethod(PostRequestForCountryCode.java:35)
  java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
  java.base/java.lang.reflect.Method.invoke(Method.java:580)
  org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
  org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
  org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
  org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
  org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
  org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
  org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
  org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
  org.testng.TestRunner.privateRun(TestRunner.java:848)
  org.testng.TestRunner.run(TestRunner.java:621)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
  org.testng.SuiteRunner.run(SuiteRunner.java:336)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
  org.testng.TestNG.runSuites(TestNG.java:1114)
  org.testng.TestNG.run(TestNG.java:1082)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
